### PR TITLE
fix most E301 warnings

### PR DESCRIPTION
**Pull request by fchapoton**
### :books: Description

This fixes most E301 warnings, except 2 problematic ones relating to strange new syntax for code typing

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/quadratic_forms/quadratic_form.py [hunk 1]
```diff
@@ -964,19 +963,22 @@ def sum_by_coefficients_with(self, right):
 #            raise TypeError, "Oh no! The multiplier cannot be coerced into the base ring of the quadratic form. =("
 #
 #        return QuadraticForm(self.base_ring(), self.dim(), [c * self.__coeffs[i]  for i in range(len(self.__coeffs))])
-# =========================================================================================================================
```
Comment (dcoudert) [inline src/sage/quadratic_forms/quadratic_form.py:967]: I don't understand this change in view of the bloc of commented code.

File: src/sage/quadratic_forms/quadratic_form.py [hunk 2]
```diff
@@ -964,19 +963,22 @@ def sum_by_coefficients_with(self, right):
...
+
+    # =================================================================
 
     def __call__(self, v):
         r"""
         Evaluate this quadratic form `Q` on a vector or matrix of elements
-        coercible to the base ring of the quadratic form.  If a vector
+        coercible to the base ring of the quadratic form.
+
+        If a vector
```
Comment (dcoudert) [inline src/sage/quadratic_forms/quadratic_form.py]: You could align this bloc in 80 columns mode

File: src/sage/schemes/elliptic_curves/ell_rational_field.py [hunk 3]
```diff
@@ -6643,6 +6643,7 @@ def reduction_at(p):
                 return (H_q)
     #<-------------------------------------------------------------------------
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/ell_rational_field.py]: Isn't it a problem to let this line without space after `#` ?

File: src/sage/schemes/elliptic_curves/ell_rational_field.py [hunk 4]
```diff
@@ -6727,6 +6728,7 @@ def test_with_T(R):
             return xs
     #<-------------------------------------------------------------------------
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/ell_rational_field.py]: same question here

**Review by dcoudert — CHANGES_REQUESTED** [review]
only a few minor remarks.

**Review by dcoudert — APPROVED** [review]
LGTM.

**Comment by fchapoton** [comment]
Merci beaucoup, David.

**Pull request merged**